src/core/profiling/endoscope_backend.h

```diff
@@ -0,0 +1,163 @@
+#include <grpc/support/sync.h>
+#include <grpc/support/time.h>
+#include <stdlib.h>
+
+#ifndef ENDOSCOPE_BACKEND_H_
+#define ENDOSCOPE_BACKEND_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Data types */
+
+#define ENDO_MUTEX_TYPE gpr_mu
```

Comment:
I previously built my endoscope server in google3, so that it can directly replace the original endoscope (to debug and compare), and I can use blaze to built and test it. Now I am rewriting a separate localized version of my endoscope server to be a part of gRPC (and remove google3 support). Those old macros are being replaced.